### ISSUE TITLE
Total training samples of flanv2

### ISSUE THREAD

**Issue by paulzyzy** [LiaoMengqi/HMoRA#3]
Hi, first of all, thanks for your responding for my last question and it is really helpful. Really appreciate it!

So I have one more question about the training setting. I saw your paper says "We limit the maximum number of training steps to 10,000, conducting evaluations every 200 steps on the validation sets of all benchmarks.". So does this mean that the total number of training sample you used is 40000 out of 10-million-sample subset of Flan v2? And the proportion you listed on the appendix is the actual proportion of the 40000 training samples you were used?

The reason why I asked is because I currently only have one a6000 ada GPU and it seems impossible to finetune the qwen on that 10-million-sample subset of Flan v2. 😭 So I just want to double check with those details with you to see if I follow your paper's detail correctly that I only need to finetune total 10000steps with batch size 4, which means that total 40000 samples are used for finetuning.

Thanks again for the clarification. If there is any details I missed or you think I should pay attention to, please point it out!!! Thank you so much for your time and help.

**Comment by LiaoMengqi** [comment]
We count multiple gradient accumulation steps as a single step, and the actual batch size used is 12. The final training dataset contains over 200,000 samples; however, we did not train for a complete epoch. To facilitate training on an A6000 GPU, the batch can be split into smaller mini-batches, though this may potentially slow down the training process. Additionally, using a smaller mini-batch size could affect the computation of auxiliary losses, and we are uncertain about the extent of its potential impact.
